src/main/java/org/swisspush/reststorage/RestStorageHandler.java

```diff
@@ -586,17 +588,35 @@ private void deleteResource(RoutingContext ctx) {
                 });
     }
 
+    private boolean checkMaxSubResourcesCount(HttpServerRequest request, int subResourcesArraySize) {
+        MultiMap headers = request.headers();
+
+        // get max expand value from request header of use the configuration value as fallback
+        Integer maxStorageExpand = getInteger(headers, MAX_EXPAND_RESOURCES_HEADER, maxStorageExpandSubresources);
+        if (maxStorageExpand < subResourcesArraySize) {
```

Comment:
Shouldn't that be less than or equal? Check would fail for `subResourcesArraySize` equals to `maxStorageExpand`